### ISSUE TITLE
slim down sidebar

### ISSUE THREAD

**Issue by gnmearacaun** [LunarVim/lunarvim.org#23]
Zoom between 133% and 200% and the sidebar expands to almost 50% of the page. The content itself seems cramped on my 13.3" screen.

**Comment by Uzaaft** [comment]
The sidebar collapses for me whenever I zoom to 150%. Does it do the same for you? I am asking since it is a while since your issue was reported.

**Comment by gnmearacaun** [comment]
For me, the sidebar still takes up half the page <= 200%. That's why it's set to open at 240% on my set up. If that's the case for you it may be that my font is set to large. Kind of annoying in general for other users perhaps

**Comment by Uzaaft** [comment]
What browser are you using? OS?

**Comment by gnmearacaun** [comment]
Thanks. I'm using Debian + LXQt. I'll attach a screenshot of my firefox (latest) settings
![Screenshot_20211217_142036](https://user-images.githubusercontent.com/23156156/146558437-a94a22a1-a70a-45a5-b1c7-d1b9ec6ac371.png)

**Comment by Uzaaft** [comment]
I tested the same font and font size in firefox on my Arch machine. I was still not able to reproduce the error.

**Comment by gnmearacaun** [comment]
Interesting. I might be something else I FF settings I can check when I get
home. Also I have my screen resolution DPI at a custom setting (which I can
advise you on later too) I plan on doing a fresh install with my black
Friday nvme SSD deal. I'll be switching to dwm. Maybe I'll should give Arch
a try this time. I've been a Debian truehead for >20 years now

**Comment by Uzaaft** [comment]
xD Well, unless you are a poweruser and customization freak I would advice you to use Manjaro. xD 
Could you try using chrome? I think Firefox has a different method for rendering compared to chrome or Safari

**Comment by gnmearacaun** [comment]
Food for thought. Manjaro + dwm vs Debian + dwm. This interests me greatly.

Yep will give it as a shot a chrome and get back to you tomorrow.

Thnx!

**Comment by Uzaaft** [comment]
Why dwm? Just wondering

**Comment by gnmearacaun** [comment]
I'm ready for dwm. I'm fed I with running out of ram. 8gb is not enough,
and I am always having to log out and log in to clear high memory usage.
Kwin is regularly using 100% of I one the cores.

**Comment by Uzaaft** [comment]
I see. xD Well, I wish you good luck on your endeavour.  I tried it, but recompiling stuff became tiring after a few days.

**Comment by gnmearacaun** [comment]
I may go, like you, with Arch. I set up a pinebook with manjaro for my GF
last year. Doesn't it come prebaked with a window manager though? Want to
know away from kwin and xcompr !

**Comment by Uzaaft** [comment]
Manjaro has different editions, which you can see here: 
https://manjaro.org/download/ and https://manjaro.org/download/#Community
If you choose KDE, you can follow the KDE guide on Arch wiki: https://wiki.archlinux.org/title/KDE
Section 5.1 mentions how to replace the WM. 
Also, if you do choose Arch, I have only one thing to say: Welcome to the cult. xD

**Comment by gnmearacaun** [comment]
That's all the encouragement I needed. Arch it is! Will be time to set this
up next couple of weeks, my support-guy 😀 Please wait a little in the
Chromium test. Won't go to it today, but looking forward to ironing it out
in the next few days 🙏

**Comment by Uzaaft** [comment]
xD Good luck! We got a pretty nice community of Linux Users in the LunarVim discord chat if you have any problems. Or you can me reach me at Discord(uzaaft#8387).

**Comment by gnmearacaun** [comment]
Excellent. It'll definitely take some adjustment. New installation + dwm +
renewed lunarvim config. I want it all done before I start a new django +
vue.js + Rest API e-commerce store project in the new year. Thanks. I
appreciate that I can draw on your support.

**Comment by Uzaaft** [comment]
I can advice abzcoding's LunarVim config xD

**Comment by gnmearacaun** [comment]
I tested this on Chromium and it's pretty much the same, but I will close this issue as it seems the problem might be unique to me. I will close the issue.

Albeit we will undoubtedly continue the discussion on Discord/Matrix, I have settled on the ArcolinuxB image with Dusk (a fork of DWM by the maker of dwm-flexipatch). Leftwm also looks interesting. 

Incidentally, my LV config is based on ABZ's. However, his changes daily! I will bring this to Matrix to see how others are playing their keybindings etc on top of his.
